vivisect/__init__.py

```diff
@@ -606,6 +611,9 @@ def analyze(self):
         Call this to ask any available analysis modules
         to do their thing...
         """
+        if self.getMeta('GUID') == None:
```

Comment:
apply the GUID when analyze() is called.  is there a preferred time for this?  we don't want to create it in VivWorkspace.__init__  because it needs to be set by loadWorkspace().  we could do it at loadFromFile and loadFromFd.  i'm open to this.  this is my only concern about this PR.